### ISSUE TITLE
Error installing Neos: Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by GustavoMarcante** [neos/neos-development-collection#1588]
Hello.

### Description

I tried to install Neos in Ubuntu 16.04 and got the error Segmentation fault (core dumped)

### Steps to Reproduce

1. I followed the instructions at http://neos.readthedocs.io/en/stable/GettingStarted/Installation.html,
installing the files at /opt/Neos.
2. I created a link /var/www//hmtl/neos pointing to /opt/Neos/Web
3. In the browser I put the address localhost/neos and returned this error (see below).
The same to variations like http://localhost/neos/index.php

I also tried to run
 ./flow server:run
at /opt/Neos and I got exactly the same error.

#### Expected behavior

Run the setup

#### Actual behavior

Both the brower and command line showed:

Segmentation fault (core dumped)
Exception Code	1355480641
Exception Type	Neos\Flow\Core\Booting\Exception\SubProcessException
Log Reference	20170529042206ed583e
Thrown in File	Packages/Framework/Neos.Flow/Classes/Core/Booting/Scripts.php
Line	611

The log is:
Exception #1355480641 in line 300 of /mnt/HDP-Dados/opt/Neos/Packages/Framework/Neos.Flow/Classes/Core/Booting/Scripts.php: Segmentation fault (core dumped)

13 Neos\Flow\Core\Booting\Scripts::executeCommand("neos.flow:core:compile", array|16|)
12 Neos\Flow\Core\Booting\Scripts::initializeProxyClasses(Neos\Flow\Core\Bootstrap)
11 call_user_func(array|2|, Neos\Flow\Core\Bootstrap)
10 Neos\Flow\Core\Booting\Step::__invoke(Neos\Flow\Core\Bootstrap)
9 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
8 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
7 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
6 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
5 Neos\Flow\Core\Booting\Sequence::invoke(Neos\Flow\Core\Bootstrap)
4 Neos\Flow\Cli\CommandRequestHandler::boot("Runtime")
3 Neos\Flow\Cli\CommandRequestHandler::handleRequest()
2 Neos\Flow\Core\Bootstrap::run()
1 require("/mnt/HDP-Dados/opt/Neos/Packages/Framework/Neos.Flow/Scripts/flow.php")

### Affected Versions

Neos: latest (neos/setup 4.0.1)
Flow: latest

Thanks!
And congratulations for the nice work!

**Comment by kitsunet** [comment]
Hey @gustavomarcante can you check if you installed igbinary and if so try to deinstall it, we had problems with it in the past. Also if you installed xdebug make sure to increase the stack size, but that usually results in a clear error message, so I doubt that's the problem.

**Comment by gustavomarcante** [comment]
Hi,

I didn't mention, but before posting my question, I tried do change in /etc/php/7.0/apache2/php.ini:
memory_limit = 256M
with no result.

I also had a base neos in MySQL and I dropped it.

After your post:
1. I checked, the package php-igbinary was installed, I removed it.
2. xdebug is not installed.
3. I restarted apache with "sudo service apache2 restart".
4. I tried again to install Neos with "./flow server:run".
And it gave a new error:

Notice: unserialize(): Error at offset 0 of 1033128 bytes in /mnt/HDP-Dados/opt/Neos/Packages/Libraries/neos/cache/Classes/Frontend/VariableFrontend.php line 91
Type: Neos\Flow\Error\Exception
  Code: 1
  File: Packages/Framework/Neos.Flow/Classes/Error/ErrorHandler.php
  Line: 81

Open Data/Logs/Exceptions/20170529142637a9b010.txt for a full stack
trace.

  Type: Neos\Flow\Core\Booting\Exception\SubProcessException
  Code: 1355480641
  File: Packages/Framework/Neos.Flow/Classes/Core/Booting/Scripts.php
  Line: 611

The two logs were:

Exception #1355480641 in line 300 of /mnt/HDP-Dados/opt/Neos/Packages/Framework/Neos.Flow/Classes/Core/Booting/Scripts.php: Notice: unserialize(): Error at offset 0 of 1033128 bytes in /mnt/HDP-Dados/opt/Neos/Packages/Libraries/neos/cache/Classes/Frontend/VariableFrontend.php line 91

  Type: Neos\Flow\Error\Exception
  Code: 1
  File: Packages/Framework/Neos.Flow/Classes/Error/ErrorHandler.php
  Line: 81

Open Data/Logs/Exceptions/20170529143441743603.txt for a full stack trace.

13 Neos\Flow\Core\Booting\Scripts::executeCommand("neos.flow:core:compile", array|16|)
12 Neos\Flow\Core\Booting\Scripts::initializeProxyClasses(Neos\Flow\Core\Bootstrap)
11 call_user_func(array|2|, Neos\Flow\Core\Bootstrap)
10 Neos\Flow\Core\Booting\Step::__invoke(Neos\Flow\Core\Bootstrap)
9 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
8 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
7 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
6 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
5 Neos\Flow\Core\Booting\Sequence::invoke(Neos\Flow\Core\Bootstrap)
4 Neos\Flow\Cli\CommandRequestHandler::boot("Runtime")
3 Neos\Flow\Cli\CommandRequestHandler::handleRequest()
2 Neos\Flow\Core\Bootstrap::run()
1 require("/mnt/HDP-Dados/opt/Neos/Packages/Framework/Neos.Flow/Scripts/flow.php")

And

Exception #1: Notice: unserialize(): Error at offset 0 of 1033128 bytes in /mnt/HDP-Dados/opt/Neos/Packages/Libraries/neos/cache/Classes/Frontend/VariableFrontend.php line 91

21 Neos\Flow\Error\ErrorHandler::handleError(8, "unserialize(): Error at offset 0 of 1033128 bytes", "/mnt/HDP-Dados/opt/Neos/Packages/Libraries/neos/cache/Classes/Frontend/VariableFrontend.php", 91, array|2|)
20 unserialize("\00\00\00classReflectionDataArrayAccess…trine\ORM\Mapping\PostRemove\D1\00>\BA\00>")
19 Neos\Cache\Frontend\VariableFrontend::get("ReflectionData")
18 Neos\Flow\Reflection\ReflectionService::loadClassReflectionCompiletimeCache()
17 Neos\Flow\Reflection\ReflectionService::initialize()
16 Neos\Flow\Reflection\ReflectionService::buildReflectionData(array|25|)
15 Neos\Flow\ObjectManagement\CompileTimeObjectManager::initialize(array|91|)
14 Neos\Flow\Core\Booting\Scripts::initializeObjectManagerCompileTimeFinalize(Neos\Flow\Core\Bootstrap)
13 call_user_func(array|2|, Neos\Flow\Core\Bootstrap)
12 Neos\Flow\Core\Booting\Step::__invoke(Neos\Flow\Core\Bootstrap)
11 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
10 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
9 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
8 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
7 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
6 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
5 Neos\Flow\Core\Booting\Sequence::invokeStep(Neos\Flow\Core\Booting\Step, Neos\Flow\Core\Bootstrap)
4 Neos\Flow\Core\Booting\Sequence::invoke(Neos\Flow\Core\Bootstrap)
3 Neos\Flow\Cli\CommandRequestHandler::boot("Compiletime")
2 Neos\Flow\Cli\CommandRequestHandler::handleRequest()
1 Neos\Flow\Core\Bootstrap::run()

I also rebooted, but nothing changed.

Thanks

**Comment by mgoldbeck** [comment]
Hi @gustavomarcante do you still have that error, or was you able to find a solution. Otherwise I can give you a list what we are installing for our ubuntu vagrant boxes.

**Comment by gustavomarcante** [comment]
Unfortunately I could not install neos.

**Comment by mrimann** [comment]
I ran into a very similar issue today and thanks to this issue found the issue. So for others that will end up here via a search engine, here's what I found out and how I solved it:

We ran some updates from Neos 3.3.x to 3.3.6 (basically just a `composer update` nothing more and nothing less) and some installations started to fail during the deployment process. So we thought Neos 3.3.6 to have some problems. But in the end, it showed that on the same day, we rolled out Redis for caching, including the package with the PHP-Redis module. Deployments on the affected servers failed - but the exact same Neos versions worked fine on other servers that did not get Redis installed.

**In my case, installing the package `php-redis` via Puppet seems to have installed the additional package `php-igbinary` "by accident".**

After removing the unwanted package, our deployments ran through again - and after restarting the PHP-FPM server via `service php7.0-fpm restart`- also the deployed websites could be accessed again.
